src/openforms/forms/api/viewsets.py

```diff
@@ -294,7 +293,94 @@ def export(self, request, *args, **kwargs):
     def perform_destroy(self, instance):
         instance._is_deleted = True
         instance.save()
-        reversion.set_comment(_("Form deleted via the API."))
+
+
+@extend_schema(
+    parameters=[
+        OpenApiParameter(
+            name="form_uuid_or_slug",
+            location=OpenApiParameter.PATH,
+            type=str,
+            description=_("Either a UUID4 or a slug identifying the form."),
+        )
+    ]
+)
+@extend_schema_view(
+    create=extend_schema(
+        summary=_("Save form version"),
+        tags=["forms"],
+        responses={status.HTTP_201_CREATED: FormVersionSerializer},
```

Comment:
This is the default serializer + HTTP status code, so shouldn't need to be specified.